public/main.js

```diff
@@ -0,0 +1,25 @@
+const model = new Model()
+const view = new View()
+
+$(`#searchBtn`).on('click',async function(){
+    const cityName = $('#textLine').val()
+    if(!cityName)
+        return
+    model.getCityData(cityName).then(val => view.renderer(val))
```

Comment:
try to be consistent. If you are using async await that use async await. Try not to mix await and `.then()`